### PR TITLE
P15b glider cals

### PR DESCRIPTION
**Pull request by cdobs**

### CODE REVIEW

File: calibration/CTDGVM/CGINS-CTDGVM-09029__20200912.csv [hunk 1]
```diff
@@ -0,0 +1,2 @@
+serial,name,value,notes
+9029,CC_calibration_date,1599868800000,sensor calibration date (milliseconds since 1970-01-01)
```
Comment (snwhite) [inline calibration/CTDGVM/CGINS-CTDGVM-09029__20200912.csv:2]: Looks good.

File: calibration/CTDGVM/CGINS-CTDGVM-09338__20170830.csv [hunk 2]
```diff
@@ -0,0 +1,2 @@
+serial,name,value,notes
+9338,CC_calibration_date,1504051200000,sensor calibration date (milliseconds since 1970-01-01)
```
Comment (snwhite) [inline calibration/CTDGVM/CGINS-CTDGVM-09338__20170830.csv:2]: Looks good.

File: calibration/CTDGVM/CGINS-CTDGVM-09338__20201001.csv [hunk 3]
```diff
@@ -0,0 +1,2 @@
+serial,name,value,notes
+9338,CC_calibration_date,1601510400000,sensor calibration date (milliseconds since 1970-01-01)
```
Comment (snwhite) [inline calibration/CTDGVM/CGINS-CTDGVM-09338__20201001.csv:2]: Looks good.

File: calibration/DOSTAM/CGINS-DOSTAM-00436__20200211.csv [hunk 4]
```diff
@@ -0,0 +1,2 @@
+serial,name,value,notes
+436,CC_calibration_date,1581379200000,sensor calibration date (milliseconds since 1970-01-01)
```
Comment (snwhite) [inline calibration/DOSTAM/CGINS-DOSTAM-00436__20200211.csv:2]: Looks good.

File: calibration/FLORTM/CGINS-FLORTM-03921__20200910.csv [hunk 5]
```diff
@@ -0,0 +1,5 @@
+serial,name,value,notes
+3921,CC_angular_resolution,1.076,constant
+3921,CC_depolarization_ratio,0.039,constant
+3921,CC_measurement_wavelength,700.0,constant
+3921,CC_scattering_angle,124.0,constant
```
Comment (snwhite) [inline calibration/FLORTM/CGINS-FLORTM-03921__20200910.csv:5]: Looks good.

File: calibration/FLORTM/CGINS-FLORTM-04472__20201014.csv [hunk 6]
```diff
@@ -0,0 +1,5 @@
+serial,name,value,notes
+4472,CC_angular_resolution,1.076,
+4472,CC_depolarization_ratio,0.039,
+4472,CC_measurement_wavelength,700,
+4472,CC_scattering_angle,124,
```
Comment (snwhite) [inline calibration/FLORTM/CGINS-FLORTM-04472__20201014.csv:5]: Looks good.

File: calibration/PARADM/CGINS-PARADM-50242__20201103.csv [hunk 7]
```diff
@@ -0,0 +1,2 @@
+serial,name,value,notes
+50242,CC_bsipar_par_scaling,1,
```
Comment (snwhite) [inline calibration/PARADM/CGINS-PARADM-50242__20201103.csv:2]: Looks good.

**Review by snwhite — APPROVED** [review]

File: calibration/CTDGVM/CGINS-CTDGVM-09029__20200912.csv (hunk 1 above)
Comment (rtravis-whoi) [reply calibration/CTDGVM/CGINS-CTDGVM-09029__20200912.csv:2]: good

File: calibration/CTDGVM/CGINS-CTDGVM-09338__20170830.csv (hunk 2 above)
Comment (rtravis-whoi) [reply calibration/CTDGVM/CGINS-CTDGVM-09338__20170830.csv:2]: good

File: calibration/CTDGVM/CGINS-CTDGVM-09338__20201001.csv (hunk 3 above)
Comment (rtravis-whoi) [reply calibration/CTDGVM/CGINS-CTDGVM-09338__20201001.csv:2]: good

File: calibration/DOSTAM/CGINS-DOSTAM-00436__20200211.csv (hunk 4 above)
Comment (rtravis-whoi) [reply calibration/DOSTAM/CGINS-DOSTAM-00436__20200211.csv:2]: good

File: calibration/FLORTM/CGINS-FLORTM-03921__20200910.csv (hunk 5 above)
Comment (rtravis-whoi) [reply calibration/FLORTM/CGINS-FLORTM-03921__20200910.csv:5]: good

File: calibration/FLORTM/CGINS-FLORTM-04472__20201014.csv (hunk 6 above)
Comment (rtravis-whoi) [reply calibration/FLORTM/CGINS-FLORTM-04472__20201014.csv:5]: good

File: calibration/PARADM/CGINS-PARADM-50145__20200914.csv [hunk 8]
```diff
@@ -0,0 +1,2 @@
+serial,name,value,notes
+50145,CC_bsipar_par_scaling,1,constant
```
[the remark this answers is not in the document]
Comment (rtravis-whoi) [reply calibration/PARADM/CGINS-PARADM-50145__20200914.csv:2]: good

File: calibration/PARADM/CGINS-PARADM-50242__20201103.csv (hunk 7 above)
Comment (rtravis-whoi) [reply calibration/PARADM/CGINS-PARADM-50242__20201103.csv:2]: good

**Review by rtravis-whoi — APPROVED** [review]

**Pull request merged**